spec/deaf_grandma_spec.rb

```diff
@@ -5,14 +5,15 @@
   let(:script) { DeafGrandma.new }
 
   it "says 'SPEAK UP SONNY!' when we speak regularly" do
-    expect(script.speak("Hi Grandma")).to eq "SPEAK UP SONNY!"
+    expect(script.speak('Hi Grandma')).to eq 'SPEAK UP SONNY!'
   end
-  
+
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak('HI GRANDMA')).to eq 'NOT SINCE 1964!'
   end
 
-  it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+  it 'EXTRA CREDIT: How would you test yelling BYE?' do
```

Comment:
Does this test pass, given that your `speak` function above uses `p 'SEE YOU LATER SONNY!'` instead of returning this string?